### DISCUSSION TITLE
Prisma migrate without running seed

### DISCUSSION THREAD

**Discussion by leo-petrucci** [prisma/prisma#8236] [Q&A] [answered]
Any way to run `prisma migrate` without also running the seed script automatically?

I'm using `prisma migrate reset` to delete my database and run a bunch of scripts to set up the development environment, but I'd rather not run the seeding at the same time.

**Comment by tomhoule** [comment, marked as answer]
There is the `--skip-seed` flag that will do that, but it's not the default. This could be configured (for example through `package.json`) though, if it's important for your use case, an issue would be great so we can track this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
